### DISCUSSION TITLE
KafkaConnector Debezium Postgres connector metrics

### DISCUSSION THREAD

**Discussion by raj-saxena** [strimzi/strimzi-kafka-operator#9118] [Q&A] [answered]
Hi,

We are using Debezium Postgres source connectors in our setup that [providers JMX metrics](https://debezium.io/documentation/reference/2.3/connectors/postgresql.html#postgresql-monitoring). We were hoping to be able to configure a `jmxPrometheusExporter` just like how we can for other Strimzi entities like `KafkaConnect` but this property doesn't exists [in the spec](https://strimzi.io/docs/operators/latest/configuring.html#type-KafkaConnectorSpec-reference). 

I believe that the property doesn't exist because not all connectors support JMX. 

Any advice on how we can export Prometheus metrics for our connectors?

If it matters, we are mostly interested in the [java.* & jvm.* metrics](https://gist.github.com/jigar1101/ba02fef8c8a8ccb7550b75e95db2abcd) to tune the resource usage.

**Comment by scholzj** [comment, marked as answer]
KafkaConnector does not result in any JVM or Pod to be started. It is just an object inside the Kafka Connect cluster. So the metrics are configured in the KafkaConnect custom resource.

**Reply by raj-saxena** [reply]
Ah, you are right. The actual pods are created from `spec.template.pod` config of `KafkaConnect`. 
Thanks for the quick reply.
